pkg/factories/rune_factory.go

```diff
@@ -242,3 +299,46 @@ func writeKafkaSource(dw printers.PrefixWriter, source *v1alpha1.KafkaSource) {
 	dw.WriteAttribute("Topics", strings.Join(source.Spec.Topics, ","))
 	dw.WriteAttribute("ConsumerGroup", source.Spec.ConsumerGroup)
 }
+
+// printKafkaSource populates a single row of kafka source list table
+func printKafkaSource(kafkaSource *v1alpha1.KafkaSource, options printers.PrintOptions) ([]metav1.TableRow, error) {
+	row := metav1.TableRow{
+		Object: runtime.RawExtension{Object: kafkaSource},
+	}
+
+	if options.AllNamespaces {
+		row.Cells = append(row.Cells, kafkaSource.ObjectMeta.Namespace)
+	}
+
+	row.Cells = append(row.Cells,
+		trunc(kafkaSource.ObjectMeta.Name),
+		commands.Age(kafkaSource.ObjectMeta.CreationTimestamp.Time),
+		trunc(kafkaSource.Spec.Sink.Ref.Name),
```

Comment:
Wrt other suggestion about sing prefix, that's how it's achieved in `kn`.
```suggestion
		trunc(flags.SinkToString(*kafkaSource.Spec.Sink)),
```

Reply:
```
NAME         AGE   SINK                 BOOTSTRAPSERVERS
mykafkasrc   13d   ksvc:event-display   my-cluster-kafka-bootstrap.kafka.svc:9092
```